packages/devtools/lib/src/charts/flame_chart_canvas.dart

```diff
@@ -33,44 +32,39 @@ const _shadedBackgroundColor =
 
 const _fontSize = 14.0;
 const _textOffsetY = 18.0;
-const _flameChartTop = rowHeightWithPadding;
-const _rowHeight = 25.0;
-const _rowPadding = 2.0;
-const rowHeightWithPadding = _rowHeight + _rowPadding;
-
-const _flameChartInset = 70;
+const rowPadding = 2.0;
+const rowHeightWithPadding = flameChartRowHeight + rowPadding;
+const flameChartTop = rowHeightWithPadding;
+const flameChartRowHeight = 25.0;
+const flameChartInset = 70;
```

Comment:
nit: drive by comment: make these consts all explicitly double even though it can be inferred. For example, the 70 probably should be a double for future consistency and to avoid int vs double asserts in the future particularly if this code is ever run on a Dart VM.